civcraftConfigs/zionconfig.yml

```diff
@@ -184,9 +184,17 @@ monster:
       despawn_on_chunk_unload: true
       maximum_light_level: 11
       drops:
-       firebones:
+       firerod:
+        material: BLAZE_ROD
+        amount: 2
+        lore: Ouch! That is Hot!
```

Comment:
some day i would like to see some factory mod recipes for useing lored items to make things for either XP or other Top Tier Factory Upgrades in the future. other than that its just a lore for a enchanted item to give it some story that has yet TBD